vertx-mysql-client/src/main/asciidoc/index.adoc

```diff
@@ -252,6 +253,17 @@ create easily create a string directly from the row set:
 {@link examples.MySQLClientExamples#collector02Example(io.vertx.sqlclient.SqlClient)}
 ----
 
+== MySQL Stored Procedure
+
+This client supports for running stored procedures in queries, the result will be retrieved from the server following the https://dev.mysql.com/doc/dev/mysql-server/8.0.12/page_protocol_command_phase_sp.html[MySQL protocol] without any magic here.
```

Comment:
rather "You can run stored procedures in queries. The result will ."

Reply:
updated in https://github.com/eclipse-vertx/vertx-sql-client/pull/403/commits/35de77b6698f39ea616f85819fd0069e46216d47.